desk/sur/contacts.hoon

```diff
@@ -89,8 +89,10 @@
 ::
 ++  get-contact
   |=  [=bowl:gall who=@p]
-  ^-  (unit contact)  ~+
-  =/  base=path  /(scot %p our.bowl)/contacts/(scot %da now.bowl)
+  =>  :_  ..get-contact
```

Comment:
what does this do?

Reply:
We want to memoize (with `~+`) the computations below. For that, we need a "fixed" subject, as it becomes the memoization key. We need some stdlib functions as well as some types from the file/core we're currently in, so instead of including those by hand with `[contact=contact ..zuse]` or similar, we just say `..get-contact` for "the context of this arm".

Reply:
interesting TIL